include/aspect/utilities.h

```diff
@@ -533,6 +533,11 @@ namespace aspect
         std::string
         get_column_name_from_index(const unsigned int column_index) const;
 
+        /**
+         * Return the maximum value of the coordinate values.
+         */
+        double get_maximum_coordinate_value() const;
```

Comment:
I do not like the name of this function. In particular, I do not quite remember how this class works, and the name of the function does not help me understand it either. 
For example in the ascii data topography case: Does the table of `AsciiDataLookup` have `dim` columns or `dim-1` coordinates and one data component, with the first `dim-1` being the coordinates and the last column being the topography? Then what about using the following declaration instead: `get_maximum_data_value(unsigned int component_index) const`? This way we would not already assume what is in the columns (because for boundary composition or boundary temperature plugins that can be different). Additionally it helps to understand what this function is actually returning, and can also be used when there is more than one data column.

Reply:
Or does this function actually return the maximum `coordinates`? But then I do not understand. The coordinates in a 2D model would be just the 1D x-coordinates, but you are interested in the maximum y-coordinate / topography, which is in the data column, not the coordinate column.

Reply:
Yes, you are right that it is probably better to do this more general way. I also noticed I put something in the wrong place, that should now be fixed. I hope it is a bit more clear now.